students/16th/johnnytae/js/login.js

```diff
@@ -0,0 +1,17 @@
+const loginButton = document.getElementsByClassName('loginButton')[0];
+const userId = document.getElementsByClassName('userId')[0];
+const userPw = document.getElementsByClassName('userPw')[0];
+
+function activateLogin() {
+  const filled = userId.value && userPw.value;
+  if (!filled) {
+    loginButton.disabled = true;
+    loginButton.style.backgroundColor = '#b5dcfc';
+  } else {
+    loginButton.disabled = false;
+    loginButton.style.backgroundColor = '#49aafa';
+  }
```

Comment:
굿! 이렇게 하실 수도 있고 다음과 같이 하실 수도 있습니다.
```suggestion
    loginButton.disabled = filled ? false: true;
    loginButton.style.backgroundColor = filled ? '#49aafa' : '#b5dcfc';
```